docs/cql/ddl.rst

```diff
@@ -897,6 +910,46 @@ The following modes are available:
+
+=============================== =============== ===================================================================================
+ Option                          Default         Description
+=============================== =============== ===================================================================================
+ ``expected_data_size_in_gb``   0               The expected table data size, in combination with the ``target_tablet_size_in_bytes``
+                                                config option, the replication factor configuration, and the cluster topology
+                                                determine the minimum number of tablets to allocate for the table.
+                                                It can be set when the table is created to allocate more tablets for it,
+                                                as if it already occupies that size.
+                                                It can also be used later in the tablet life cycle to induce tablet splits or merges
+                                                to fit the new expected size.
+ ``hot_table``                  'false'         Hint that this table expects relatively higher traffic than other tables so that the
+                                                tablet allocation algorithm should give it higher priority for balancing its tablets.
+                                                For example, the automatic algorithm may decide to allocate more tablets for this table
+                                                to reduce the imbalance between nodes.
+                                                Note: Not implemented yet (option value is ignored).
+ ``min_per_shard_tablet_count`` 0               Used for ensuring that the table workload is well balanced in the whole cluster in a
+                                                topology- independent way. A higher number of tablet replicas per shard may help balance
+                                                the table workload more evenly across shards and across nodes in the cluster.
+                                                For example, setting this to 10 means that shard overcommit is limited to 10%, regardless
+                                                of cluster size.
+ ``min_tablet_count``           0               Determines the minimum number of tablets to allocate for the table.
+                                                The hint is backward compatible with the deprecated keyspace ``initial`` tablets option.
+                                                Note that the actual number of tablet replicas that are owned by each shard are a
+                                                function of the tablet count, the replication factor in the datacenter, and the number
+                                                of nodes their shards in the datacanter.  It is recommended to use higher-level hints
```

Comment:
of nodes their shards -> of nodes and shards ?

Reply:
np

Reply:
fixed in v2